EnemizerCLI.Core/Program.cs

```diff
@@ -13,27 +13,27 @@ class Program
     {
         static void Main(string[] args)
         {
-            var options = new CommandLineOptions();
-            if (CommandLine.Parser.Default.ParseArgumentsStrict(args, options))
+	        var writer = new StringWriter();
+            var parser = new Parser(conf => conf.HelpWriter = writer);
+            var result = parser.ParseArguments<CommandLineOptions>(args);
+
+            result.WithNotParsed(error =>
             {
-                Stopwatch stopwatch = new Stopwatch();
-                if (options.BinaryMode)
-				{
-                    stopwatch.Start();
-                    MakeEnemizerRom(options);
-				}
-                else
-				{
-                    stopwatch.Start();
-                    MakeEnemizerJsonPatch(options);
-                }
-                stopwatch.Stop();
-                Console.WriteLine($"Seed generated in: {stopwatch.Elapsed}");
-            }
-            else
+				Console.WriteLine(writer.ToString());
+				Environment.Exit(1);
+            }).WithParsed(options =>
             {
-                Console.Write(GetUsage(options));
-            }
+				Stopwatch stopwatch = new Stopwatch();
+				stopwatch.Start();
+
+	            if (options.BinaryMode)
+		            MakeEnemizerRom(options);
+	            else
+		            MakeEnemizerJsonPatch(options);
```

Comment:
This is minor, but it's a code style I stick with always. Please use braces for all control flow statements, even one-liners.

Reply:
Okay, will keep that in mind with this project going forward.